docs/book/features/pv_to_backingdiskobjectid_mapping.md

```diff
@@ -0,0 +1,93 @@
+<!-- markdownlint-disable MD033 -->
+<!-- markdownlint-disable MD034 -->
+# PV to BackingDiskObjectId Mapping
+
+- [Introduction](#introduction)
+- [Prerequisite](#prereq)
+- [How to enable PV to BackingDiskObjectId Mapping feature in vSphere CSI](#how-to-enable)
+- [How to use PV to BackingDiskObjectId Mapping feature](#how-to-use)
+
+## Introduction <a id="introduction"></a>
+
+PV to BackingDiskObjectId Mapping feature will be added in vSphere CSI driver 2.5 as an Alpha feature. This feature provide backup partners with PV to BackingDiskObjectId mapping
+so they can do volume placement on the target side.
+
+Known limitations for the Alpha feature in vSphere CSI driver 2.5 are listed below.
+
+1. Retrieving backingDiskObjectId is only supported for vVOl and vSAN, so it is not supported for PVs that are provisioned through other types.
+2. It is only supported in vanilla cluster, currently it will not be supported in Supervisor/Guest cluster. (will supported in the future)
+
+## Prerequisite <a id="prereq"></a>
+
+- vSphere version required is 7.0 U2 or above.
+
+## How to enable PV to BackingDiskObjectId Mapping feature in vSphere CSI <a id="how-to-enable"></a>
+
+- To enable PV to BackingDiskObjectId Mapping feature, patch the configmap to enable `pv-to-backingdiskobjectid-mapping` feature switch by running following command:
+  
+  ```bash
+  $ kubectl patch configmap/internal-feature-states.csi.vsphere.vmware.com \
+  -n vmware-system-csi \
+  --type merge \
+  -p '{"data":{"pv-to-backingdiskobjectid-mapping":"true"}}'
+  ```
+  
+  Verify that `pv-to-backingdiskobjectid-mapping` is enabled:
+  
+  ```bash
+  # kubectl edit configmap internal-feature-states.csi.vsphere.vmware.com -n vmware-system-csi
+  apiVersion: v1
+  data:
+    ...
+    pv-to-backingdiskobjectid-mapping: "true"
+    ...
+  ```
+  
+## How to use PV to BackingDiskObjectId Mapping feature <a id="how-to-use"></a>
+
+- After `pv-to-backingdiskobjectid-mapping` feature is enabled, restart vsphere-csi-controller pod.
+
+  ```bash
+  kubectl -n vmware-system-csi scale deploy/vsphere-csi-controller --replicas=0
+  kubectl -n vmware-system-csi scale deploy/vsphere-csi-controller --replicas=1
+  ```
+
+- Below is the expected view in `vmware-system-csi` namespace when the deployment is completed in a single-master cluster:
```

Comment:
I think this line should be modified a little bit and moved before line 50 as you used replicas=1 on line 52.